Practical-4

gauravMori158 opened the pull request:
Inside Practical - 4

gauravMori158 committed:
Practical-4

Inside Practical - 4

sagar-vaja-simform commented on Practical - 4/Student.cs:
```diff
@@ -0,0 +1,180 @@
+
+            foreach (decimal mark in marks)
+            {
+                if (mark > max) max = mark;
```
use Max method of array

sagar-vaja-simform commented on Practical - 4/Student.cs:
```diff
@@ -0,0 +1,180 @@
+
+            foreach (decimal mark in marks)
+            {
+                if (mark < min) min = mark;
```
Use min method of array

sagar-vaja-simform commented on Practical - 4/Student.cs:
```diff
@@ -0,0 +1,180 @@
+
+            for (int i = 0; i < 5; i++)
+            {
+                student.Marks[i] = Convert.ToDecimal(Console.ReadLine());
```
Handle exception

sagar-vaja-simform commented on Practical - 4/Student.cs:
```diff
@@ -0,0 +1,180 @@
+            while (true)
+            {
+
+                int option = Convert.ToInt32(Console.ReadLine());
```
Handle exception

gauravMori158 committed:
Update Student.cs

Used Max, Min and Average Method for array operations.

Handle Exception :  1. IndexOutOfRangeException
and int.tryPrase
